src/test/kotlin/StringCalculatorTest.kt

```diff
@@ -38,22 +39,22 @@ class StringCalculatorTest {
         }
 
         private fun validateNumber(input: String) {
-            if(input.toIntOrNull() == null) {
+            if (input.toIntOrNull() == null) {
                 throw IllegalArgumentException("Enter a valid expression")
             }
         }
 
-        private fun apply(result: Int, sign: String, targetNumber: Int): Int{
-            if(sign == "+") {
+        private fun apply(result: Int, sign: String, targetNumber: Int): Int {
+            if (sign == "+") {
                 return result + targetNumber
             }
-            if(sign == "-") {
+            if (sign == "-") {
                 return result - targetNumber
             }
-            if(sign == "*") {
+            if (sign == "*") {
                 return result * targetNumber
             }
-            if(sign == "/") {
+            if (sign == "/") {
```

Comment:
If you have time, you could refactor your code using an enum class.

Reply:
yeap I will do it. :)